geometry_msgs/Pose2D start_pose
geometry_msgs/Pose2D end_pose
---
std_msgs/Bool finished